contracts/ZkVerifyAggregationGlobal.sol

```diff
@@ -2,14 +2,118 @@
 
 pragma solidity 0.8.20;
 
+import "@openzeppelin/contracts-upgradeable/access/AccessControlUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+import "@openzeppelin/contracts-upgradeable/proxy/utils/UUPSUpgradeable.sol";
 import "./ZkVerifyAggregationIsmp.sol";
 import "./ZkVerifyAggregation.sol";
 
 /**
  * @title ZkVerifyAggregationGlobal Contract
  * @notice It allows submitting and verifying aggregation proofs coming from zkVerify chain for both versions Ismp and Non Ismp one.
  */
-contract ZkVerifyAggregationGlobal is ZkVerifyAggregation, ZkVerifyAggregationIsmp {
+contract ZkVerifyAggregationGlobal is ZkVerifyAggregationBase, Initializable, AccessControlUpgradeable, UUPSUpgradeable, IsmpGuest, BaseIsmpModule {
```

Comment:
I would inherit from `ZkVerifyAggregationBase` last. That has to do with how Solidity assigns storage slots to inherithed contracts ([official docs](https://docs.soliditylang.org/en/v0.8.14/internals/layout_in_storage.html#layout-of-state-variables-in-storage) and [hands-on example](https://ethereum.stackexchange.com/questions/63403/in-solidity-how-does-the-slot-assignation-work-for-storage-variables-when-there)). If we don't do like this, then doing updates to the storage layout of `ZkVerifyAggregationBase` without causing storage collisions would be tricky.
```suggestion
contract ZkVerifyAggregationGlobal is Initializable, AccessControlUpgradeable, UUPSUpgradeable, IsmpGuest, BaseIsmpModule, ZkVerifyAggregationBase {
```